source/gui/nvdaControls.py

```diff
@@ -42,6 +42,41 @@ def OnSetFocus(self, evt):
 		self.SetSelection(0, numChars)
 		evt.Skip()
 
+class EnhancedInputSlider(wx.Slider):

```

Comment:
Is this class added or moved from elsewhere? What does it do exactly? Might need a docstring.

Reply:
This is a rename from gui.settingsDialogs.VoiceSettingsSlider, and I really do not know why it even exists.